src/ORM/Association/BelongsToMany.php

```diff
@@ -673,8 +673,8 @@ protected function _saveLinks(EntityInterface $sourceEntity, $targetEntities, $o
     /**
      * Associates the source entity to each of the target entities provided by
      * creating links in the junction table. Both the source entity and each of
-     * the target entities are assumed to be already persisted, if the are marked
-     * as new or their status is unknown, an exception will be thrown.
+     * the target entities are assumed to be already persisted, if they are marked
+     * as new or their status is unknown then an exception will be thrown.
```

Comment:
Grammar can be very locale driven. 

- some places keep the last comma and others drop it (in Canada it is dropped).
- the word `an` should not follow a comma
- the word `if` should not follow a comma, but it should also not start a sentence.

We're getting into the dirty bits of grammar that I hate. I just wanted to fix `the`=`they`. The other grammar issues require some kind of general Cake guidelines on which locale the project follows for grammar. Otherwise, we'll have people arguing grammar from their locale.